core/embed/rust/src/ui/model_mercury/component/button.rs

```diff
@@ -660,4 +622,48 @@ impl IconText {
             );
         }
     }
+    pub fn render<'s>(
+        & self,
+        target: &mut impl Renderer<'s>,
+        area: Rect,
+        style: &ButtonStyle,
+        baseline_offset: Offset,
+    ) {
+        let width = style.font.text_width(self.text.as_ref());
+        let height = style.font.text_height();
```

Comment:
Unused? Also bunch of other variables in the file.

Reply:
https://github.com/trezor/trezor-firmware/pull/3599/commits/5664655e7aaf7b92db44c1e60bac06e9d3b56df0